resources/local/chainspec.toml.in

```diff
@@ -65,6 +65,8 @@ maximum_round_exponent = 19
 # The factor by which rewards for a round are multiplied if the greatest summit has ≤50% quorum, i.e. no finality.
 # Expressed as a fraction (1/5 by default).
 reduced_reward_multiplier = [1, 5]
+# The folder in which the files with per-era latest unit hashes will be stored.
+unit_hashes_folder = "../node-storage"
```

Comment:
It _shouldn't_ be in the chainspec though, but the node configuration.

Reply:
Node configuration isn't easily accessible from `Highway` :grimacing: It is in the config already, but Highway can't reach it without a lot of changes.

Reply:
Ah, right… we'll need to hand this down into `HighwayProtocol` somehow.
Not sure if having it in the chainspec temporarily breaks anything? :confused: 
It would certainly be better to fix it.

Reply:
Actually fixed it! :tada: :+1: